src/lib/rpc/xdr.c

```diff
@@ -607,7 +607,7 @@ xdr_union(
 bool_t
 xdr_string(XDR *xdrs, char **cpp, u_int maxsize)
 {
-	register char *sp = *cpp;  /* sp is the actual string pointer */
+	char *sp = *cpp;  	/* sp is the actual string pointer */
```

Comment:
Looks like unintentional whitespace change?

Reply:
Inline comments are supposed to start at column 32, I think in the Solaris style as well as ours.  However, having two spaces before the tab is anomalous even in a file which uses tabs.  I changed it to be just a tab.